packages/styled-system/src/StyledSystemProvider.tsx

```diff
@@ -1,10 +1,13 @@
-import type { ComponentProps, ReactNode } from "react";
+import { useCallback, useState, type ComponentProps, type ReactNode } from "react";
+import { ColorSchemeContext, type ColorScheme, type ColorSchemeContextType, type ColorSchemeOrSystem } from "./color-scheme/ColorSchemeContext.ts";
+import { useColorScheme } from "./color-scheme/useColorScheme.ts";
 import { BodyStyleProvider } from "./global-styles/BodyStyleProvider.tsx";
+import { BreakpointProvider, DefaultUnsupportedMatchMediaBreakpoint, type BreakpointProviderProps } from "./responsive/BreakpointProvider.tsx";
 import { HopperRootCssClass, StyledSystemRootCssClass } from "./styled-system-root-css-class.ts";
 import { TokenProvider } from "./tokens/TokenProvider.tsx";
 import clsx from "./utils/clsx.ts";
```

Comment:
ClassName management is handled by `classnames` in the documentation and certainly in hopper, so why not make use of it!

Reply:
i'm incline to add this dependency as well. I'll add https://www.npmjs.com/package/clsx instead of classnames, as it seems like the most recent alternative, and more used (~15M clsx, ~14M  classnames)

> serves as a [faster](https://github.com/lukeed/clsx/blob/HEAD/bench) & smaller drop-in replacement for the classnames module.